src/transformStateWithClones.js

```diff
@@ -65,7 +65,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
-}
+  let currentObject = Object.assign({}, state);
+
+  const reTransforms = transforms;
```

Comment:
![image](https://user-images.githubusercontent.com/38065883/89658121-ccdf2b00-d8d6-11ea-8f9c-a6405a1aae26.png)
do you believe that this is different objects?